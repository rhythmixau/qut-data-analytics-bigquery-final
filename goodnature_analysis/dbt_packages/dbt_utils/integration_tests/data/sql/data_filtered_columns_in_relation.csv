version https://git-lfs.github.com/spec/v1
oid sha256:f2cc1bd682a348241a2679bc888e7281490fccc6f9bca1aaa7aec17d3e203e09
size 41
