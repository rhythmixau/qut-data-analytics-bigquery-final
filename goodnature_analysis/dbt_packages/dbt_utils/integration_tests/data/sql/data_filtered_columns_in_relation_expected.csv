version https://git-lfs.github.com/spec/v1
oid sha256:9238b04244c715d7bfdaf9e9bd73ab255ad00de28e5724bbc739b55d4fa69f83
size 19
